R/get_metric_description.R

```diff
@@ -0,0 +1,54 @@
+#' Gets the description for a given COVID-19 metric 
+#' 
+#' Given a COVID-19 metric collected in the scraped data, returns a description of 
+#' that metric, primarily useful for plotting (e.g. axis labels, titles, etc.). 
+#'
+#' @param metric character string of the given metric 
+#' @param short logical, whether to return the short (vs. long) description 
+#' 
+#' @return character string of the given metric's description 
+#' 
+#' @examples
+#' get_metric_description("Residents.Active")
+#' 
+#' @importFrom stringr str_c
+#'
+#' @export
+
+get_metric_description <- function(metric, short = FALSE) {
+    short_lookup <- c(
+        "Residents.Active" = "Active COVID-19 Cases", 
+        "Staff.Active" = "Active COVID-19 Cases", 
+        "Residents.Confirmed" = "Cumulative COVID-19 Cases", 
+        "Staff.Confirmed" = "Cumulative COVID-19 Cases", 
+        "Residents.Deaths" = "Cumulative COVID-19 Deaths", 
+        "Staff.Deaths" = "Cumulative COVID-19 Deaths", 
+        "Residents.Tadmin" = "COVID-19 Tests" 
+    )
```

Comment:
Note to self: add description for ambiguous resident tests, present in historical data. This is called `Residents.Tested`